javers-core/src/main/java/org/javers/guava/multimap/MultimapChange.java

```diff
@@ -0,0 +1,16 @@
+package org.javers.guava.multimap;
+
+import org.javers.core.diff.changetype.map.EntryChange;
+import org.javers.core.diff.changetype.map.MapChange;
+import org.javers.core.metamodel.object.GlobalId;
+
+import java.util.List;
+
+/**
+ * @author akrystian
+ */
+public class MultimapChange extends MapChange{
```

Comment:
To zostało tak zrobione dlatego, że semantyka MultiMapy i MultiSetu jest trochę inna niż Mapy i Setu. To może być mylące w pewnych przypadkach. Ale mogę to zmienić.